config/locales/hyrax.en.yml

```diff
@@ -444,7 +444,7 @@ en:
         label: "Search Collection %{title}"
         placeholder: "Search Collection"
       show:
-        works_in_collection: "Works in this Collection"
+        works_in_collection: "Collection Items"
```

Comment:
Are you going to update the other translations for this change?

Reply:
I wasn't aware of needing to. I changed the text to match what was in the mockups provided in the issue I was assigned. If the translations are available, I can add them in no problem.